_sass/styles.scss

```diff
@@ -73,6 +73,11 @@
 //
 // Custom styles
 // ==========================================================================
+
+// Global element styles
+@import "global-elements";
```

Comment:
5. There aren't any real themes for `figures` or `captions`. So I created a new partial where we can set styles for global components.

⚠️ Be aware, you should use these sparingly and try to rely on classes as much as possible. This helps avoid conflicts.